test/getJwk.spec.js

```diff
@@ -0,0 +1,159 @@
+'use strict'
+
+const nock = require('nock')
+const t = require('tap')
+
+const buildGetJwks = require('../src/get-jwks')
+
+t.beforeEach((done, t) => {
```

Comment:
the t argument in the callback is unused, same at line 13

Reply:
Thanks for spotting that